### DISCUSSION TITLE
Shouldn't the "unstructured media" be labeled with the doctype?

### DISCUSSION THREAD

**Discussion by matthias-ronge** [kitodo/kitodo-production#4402] [General]
In the example: Shouldn't there be "monograph" instead of "unstructured media"?

![Screenshot](https://user-images.githubusercontent.com/3040657/95463471-11d81a00-0979-11eb-98c8-07cb2276574c.png)

**Comment by andre-hohmann** [comment]
@matthias-ronge : 
Logically, your suggestion is correct. The label of the structure element should be shown, to which the media files are directly assigned to. 
In my opinion, "unstructured media" is practically helpful for the users to identify quickly the media files which might need to be assigned to a structure element.  

I am undecided and can live with both variants. More opinions are needed to initiate a change. If there is no further support for a renaming, it is a sign to me that the current label "unstructured media" should be retained.

**Reply by aetherfaerber** [reply]
> @matthias-ronge : Logically, your suggestion is correct. The label of the structure element should be shown, to which the media files are directly assigned to. In my opinion, "unstructured media" is practically helpful for the users to identify quickly the media files which might need to be assigned to a structure element.

In my opinion labeling these files as “unstructured media“ is, as implemented now, rather confusing than helpful. Currently, to a newcomer It is not clear from the tree-view on the leftmost panel that the files are actually assigned to the structure element of the doctype. Changing the label in gallery panel would change that, as adding a dotted line from the parent elements in the structure tree to the child elements as well.

Additionally, having the images labeled as “unstructured media“ does suggest that the images are actually are not assigned to any structural element. Imo this leads to the conclusion that they **need to be assigned** to a structure element and not as you stated that they “might need to be assigned to a structure element”.

> I am undecided and can live with both variants. More opinions are needed to initiate a change. If there is no further support for a renaming, it is a sign to me that the current label "unstructured media" should be retained.

For me this is heavily dependent on the distinction I just pointed out. If the images are in need of being structured, labeling them as unstructured should be the way to go. If there is no need, this (assigning individual pages to structural elements) is just one possible use case (besides i.e. editing the metadata) of the metadata editor. It fully depends on what an institution's workflow expects. So maybe this should be a switch in the workflow editor for tasks that include the metadata editor?
Anyway, for now i would rather vote in favor of renaming the label to the doctype. But I also suspect that the current design and label actually assumes, that unstructured items **need** to be structured.


Besides the suggestion regarding a dotted line between parents and children in the structure tree I see room for other possible UI improvements here

1. Strengthen the connection between structure tree and gallery panel.
Currently (in the example of the screenshot above), the „Kapitel“ and „untstrukturierte Medien“ look identical, even though (as pointed out above) actually the structural element „Kapitel“ is on the same level not as the structural element „Monographie“ but as the objects/images contained therein. This could either be shown through an indentation (as is already the case in the structure tree), by placing the frame around „Kapitel“ inside the frame „unstrukturierte Medien“ (of course this would only make sense if it is renamed) or by making the frames color a bit lighter for each level down the hierarchy.

2. Put visible checkmarks on the images for metadata tasks
If assigning the images to structural elements or adding a certain metadata entries to each image is a task that needs to be done in the metadata editor, there should be a clue as to which images have been already dealt with and which are in need of treatment. For the task of assigning a structural element, this is currently done through having images without assignment reside in the „unstructured media“ frame at the bottom. It seems quite hard to repurpose this visual clue for  other tasks like assigning values in certain metadata fields. It would be easier to have several colored dot at the buttom of each image that each mark a missing but required metadata field entry. An option to assign colors to metadata in the configuration would be needed, as well as a way to filter the gallery for all items lacking an entry for a certain field.

**Reply by andre-hohmann** [reply]
@aetherfaerber : Thanks a lot for the extensive reply. I can follow the argumentation and it seems to me that the label "unstrukturierte Medien" should be replaced by the label of the docType. 

Maybe, your other suggestions should be discussed in separate discussion(s), due to the following reasons:
1. the discussions here might get confusing
2. your suggestions might not be easily recognized/found under the label "Shouldn't the "unstructured media" be labeled with the doctype?"
3. it will be difficult to close a discussion of a specific object until all other aspects are finished
4. it will be easier to link from specific issues to a discussion, in case that the suggestions will be realized
